dali/util/std_cufile.cc

```diff
@@ -115,6 +115,36 @@ void StdCUFileStream::HandleIOError(int64 ret) const {
   }
 }
 
+size_t StdCUFileStream::ReadGPUThread(uint8_t* gpu_buffer, size_t n_bytes, \
```

Comment:
I see that a lot of the code in `ReadGPUThread` is duplicated from `ReadGPU`. Is there a chance we can reuse?

Reply:
read gpu could call read gpu thread in a trivial manner

Reply:
I fixed it, please review